packages/web/docs/src/app/docs/[[...mdxPath]]/page.tsx

```diff
@@ -13,12 +14,20 @@ export async function generateMetadata(
 ) {
   const { mdxPath } = await props.params;
   const { metadata } = await importPage(mdxPath);
-  return {
+  const docsMetadata = {
     ...metadata,
     ...(mdxPath?.[0] === 'gateway' && {
       title: { absolute: `${metadata.title} | Hive Gateway` },
     }),
   };
+
+  // TODO: Remove this when Components have a fix for OG Images with basePath
+  docsMetadata.openGraph = {
+    ...rootMetadata.openGraph,
+    ...docsMetadata.openGraph,
+  };
```

Comment:
I'd have expected it to be inherited from the root layout, but it seems it's not.